src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundUserAcceptanceTest.java

```diff
@@ -82,52 +103,79 @@ void testSendEdifactIsProcessedAndPushedToGpOutboundQueue()
+        } else {
+            assertTrue(getMeshClient().getInboxMessageIds().isEmpty());
+        }
+
     }
 
-    private void assertOutboundNhsAckMessage()
-        throws IOException, InterchangeParsingException, MessageParsingException {
-
-        final var labResultMeshClient = getLabResultsMeshClient();
-        final var edifactParser = getEdifactParser();
-        final var nhsAck = new String(Files.readAllBytes(getNhsAckResource().getFile().toPath()));
-
-        // Acting as a lab results system, receive and validate the NHSACK returned by the adaptor.
-        final List<String> messageIds = waitFor(() -> {
-            final List<String> inboxMessageIds = labResultMeshClient.getInboxMessageIds();
-            return inboxMessageIds.isEmpty() ? null : inboxMessageIds;
-        });
-        var meshMessage = labResultMeshClient.getEdifactMessage(messageIds.get(0));
-
-        //TODO NIAD-851 UAT framework
-//        Interchange expectedNhsAck = edifactParser.parse(nhsAck);
-//        Interchange actualNhsAck = edifactParser.parse(meshMessage.getContent());
-//
-//        assertThat(meshMessage.getWorkflowId())
-//            .isEqualTo(WorkflowId.PATHOLOGY_ACK);
-//        assertThat(actualNhsAck.getInterchangeHeader().getRecipient())
-//            .isEqualTo(expectedNhsAck.getInterchangeHeader().getRecipient());
-//        assertThat(actualNhsAck.getInterchangeHeader().getSender())
-//            .isEqualTo(expectedNhsAck.getInterchangeHeader().getSender());
-//        assertThat(actualNhsAck.getInterchangeHeader().getSequenceNumber())
-//            .isEqualTo(expectedNhsAck.getInterchangeHeader().getSequenceNumber());
-//        assertThat(filterTimestampedSegments(actualNhsAck))
-//            .containsExactlyElementsOf(filterTimestampedSegments(expectedNhsAck));
-//        assertThat(actualNhsAck.getInterchangeTrailer().getNumberOfMessages())
-//            .isEqualTo(expectedNhsAck.getInterchangeTrailer().getNumberOfMessages());
-//        assertThat(actualNhsAck.getInterchangeTrailer().getSequenceNumber())
-//            .isEqualTo(expectedNhsAck.getInterchangeTrailer().getSequenceNumber());
+    /**
+     * The test EDIFACT message (.dat file) is sent to the MESH mailbox where the adaptor receives inbound messages.
+     * The test waits for the messages to be processed confirms that the outbound queue is empty.
+     */
+    @ParameterizedTest(name = "[{index}] - {0}")
+    @ArgumentsSource(FailureArgumentsProvider.class)
+    void testEdifactFailsToBeProcessedAndNothingPushedToGpOutboundQueue(String category, TestData testData) {
+
+        String recipient;
+        try {
+            recipient = getEdifactParser().parse(testData.getEdifact())
+                .getInterchangeHeader().getRecipient();
+        } catch (InterchangeParsingException e) {
+            recipient = e.getRecipient();
+        } catch (MessageParsingException e) {
+            recipient = e.getRecipient();
+        }
```

Comment:
Since the bodies are the same, these catches could be merged if we make `InterchangeParsingException` and `MessageParsingException` share some abstraction. They have the same contents already.

```suggestion
        } catch (InterchangeParsingException | MessageParsingException e) {
            recipient = e.getRecipient();
        } 
```

Reply:
I agree, although that would probably be outside the scope of this ticket. Create a new ticket for this possibly?

Reply:
How come outside of the scope?

Reply:
From my perspective, refactoring a part of the actual adaptor code might be better split into a separate ticket, to keep tickets and PRs focused. If you disagree though, I'm happy to make those changes in this PR.

Reply:
Isn't this your code? Anyway, it's a small change to the same file you are editing. Please apply

Reply:
I had a talk with Andrew and now I understand. Those exceptions have different base class so it's not just that simple change here but would involve much refactor on the exception classes side. 
I agree to leave as is for now